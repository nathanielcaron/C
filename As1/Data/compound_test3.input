e
e
o
u 5
e
o
u 10000
d
p
u 11100
u 10010
u 201
u -10
e
o
e
x
u 10
u 15
u 20
u 25
u 30
o
o
u 1
u 2
u 3
e
o
e
x
