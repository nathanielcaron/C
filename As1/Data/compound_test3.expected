failed peek
failed peek
failed pop
5
5
5
10000
invalid instruction d
invalid instruction p
11100
10010
201
-10
-10
-10
201
Successfully executed 11 instructions
|_ 10000 _|
|_ 11100 _|
|_ 10010 _|
|_ 201 _|
4 elements
